### PR TITLE
https://github.com/rnshah9/mayhem-smallerc

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Build Stage
+FROM --platform=linux/amd64 ubuntu:20.04
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y make gcc nasm
+
+ADD . /SmallerC
```
Comment (xansec) [inline Dockerfile:6]: Please use stages.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+project: rnshah9/mayhem-smallerc
```
Comment (xansec) [inline Mayhemfile]: Please use generic project name and correct filepath directive.

**Review by xansec — CHANGES_REQUESTED** [review]

**Comment by xansec** [comment]
Unrelated, but why is this named `dbcc`?

**Comment by rnshah9** [comment]
Not sure what happened there, must have made a typo

**Comment by xansec** [comment]
LGTM

**Pull request merged**